project/frontend/src/components/PreferenceForm.js

```diff
@@ -128,18 +145,40 @@ function PreferenceForm(props) {
   };
 
   const diningExperiences = {
-    Takeout: 'meal_takeaway',
-    Delivery: 'meal_delivery',
+    'Takeout': 'meal_takeaway',
+    'Delivery': 'meal_delivery',
     'Dine In': 'restaurant',
   };
 
   const prices = {
-    Low: 1,
-    Medium: 2,
-    High: 3,
+    'Low': 1,
+    'Medium': 2,
+    'High': 3,
     'Very High': 4,
   };
 
+  const tooltipInfo = 
+  (<React.Fragment>
+    <p>We will use your preferences entered here to find a restaurant we think you'd like.</p>
+    <p>You may leave any field blank if you have no preference for that field. If you are looking for a 
```

Comment:
i think you could do a line break `<br>` instead of having a few different <p> elements unless there's a reason these need to be separated.

Reply:
Although it would visually look the same, I used different paragraphs because they're like separate thoughts, whereas a line break is to manually break up a line within a paragraph... see https://stackoverflow.com/questions/13688158/when-to-use-p-vs-br#:~:text=You%20want%20to%20use%20the,of%20information%20into%20separate%20thoughts.&text=The%20tag%20is,line%2C%20such%20as%20with%20poetry.&text=Hello%20World!